deploy/charts/external-secrets/templates/webhook-certificate.yaml

```diff
@@ -13,13 +23,15 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
-  commonName: {{ include "external-secrets.fullname" . }}-webhook
+  commonName: "{{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}.svc"
   dnsNames:
-    - {{ include "external-secrets.fullname" . }}-webhook
-    - {{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}
-    - {{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}.svc
+    - "{{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}.svc"
+  {{- with .Values.webhook.certManager.cert.issuerRef }}
   issuerRef:
-    {{- toYaml .Values.webhook.certManager.cert.issuerRef | nindent 4 }}
+    group: {{ .group }}
+    kind: {{ .kind }}
```

Comment:
with the creation of an issuer, does it really make sense to have `kind` defined here? We`re not creating `ClusterIssuer` nor providing a path for it.

Reply:
There is a config for it, and changing that would indeed be breaking. And if you are not using a self-signed issuer (ref. your other comment), you will probably use a ClusterIssuer.